src/index.html

```diff
@@ -5,9 +5,7 @@
   <title>AngularElectron</title>
   <base href="">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="icon" type="image/x-icon" href="favicon.ico">
```

Comment:
this was here since before xel was integrated, i think its the app's tab icon or something. i think we should keep it...